### PR TITLE
feat: creating blogger class to post articles to respective platforms

### PR DESCRIPTION
**Pull request by Souvikns**
The aim of this PR is to introduce the logic to post articles to different platforms. I am creating a separate class for this because in the future when we need to add or update this logic it would be easy to do so.

### CODE REVIEW

**Comment by Souvikns** [comment]
@uzair-ali10 do you have any idea about testing the dev.to API to see it is working as intended

**Comment by uzair-ali10** [comment]
> @uzair-ali10 do you have any idea about testing the dev.to API to see it is working as intended

Yea I Can help you out with it
Please point out what you need help with

File: src/blogger.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            if(!this.dev_token) return;
+
+            const response = await Axios.post('', article, {
+                headers: {
+                    'Content-Type': 'application/json',
+                    'api-key': this.dev_token
+                }
+            });
+
+        },
```
Comment (Souvikns) [inline src/blogger.ts:25]: So this is where we will call dev.to API, yeah I am missing the URL still wip, but I don't think there is any way to write tests for it, I have doubt that if I am doing the right thing with the API.

File: src/app.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+export class App {
+    private readonly notionAdapter: NotionAdapter;
+    constructor(config: AppConstructor){
+        this.notionAdapter = NotionAdapter.instantiate({
+            api_key: config.notion.api_key,
+            database_id: config.notion.database_id
+        });
+    }
+}
```
Comment (Souvikns) [inline src/app.ts:22]: We have to figure out the interface of the library

**Comment by Souvikns** [comment]
@uzair-ali10 needs your review, I have written the general code and now I guess all it is left is to refactor it.

File: src/app.ts (hunk 2 above)
Comment (uzair-ali10) [reply src/app.ts:22]: This is still not resolved I believe, correct??

**Review by uzair-ali10 — COMMENTED** [review]
`index.js` is an empty file, i belive it would be better to remove it

**Comment by Souvikns** [comment]
I just created the `app.ts` file what I have in mind is that `runner.ts` is the script that the gh action will use and `app.ts` will have the class that we can export as a library for you to use.

File: src/app.ts (hunk 2 above)
Comment (Souvikns) [reply src/app.ts:22]: No I am still thinking of a better UX for the library open to any suggestions.

**Review by uzair-ali10 — APPROVED** [review]
Alright everything else LGTM

**Comment by Souvikns** [comment]
Is the hashnode API call fine

**Pull request merged**